dataminer-overview/DataMiner_Cloud_Platform/Overview_DCP.md

```diff
@@ -4,107 +4,189 @@ uid: Overview_DCP
+> - [DIS User Guide](https://docs.dataminer.services/develop/TOOLS/DIS/DIS.html)
 
-Skyline provides a full-fledged and fully integrated cloud-based collaboration ecosystem for project squads, named DataMiner Project Collaboration. It allows the squads, which typically consist of both Skyline staff and users of a DataMiner Solution, to manage the entire design and deployment process of the solution.
+### CI/CD
 
-When a DataMiner Solution is ordered, the intake process will train and prepare the users of the solution for the project. As part of this, they will be provided access to the Project Collaboration module.
+**Continuous Integration/Continuous Deployment (CI/CD) pipeline** is there to automating the process of building and testing code, with a focus on utilizing tools such as DIS, GitHub, GitLab, Jenkins pipeline, and others.
 
-![Project Collaboration – Board view](~/dataminer-overview/images/CC_collaboration_2.png)
+The **goal** is to test, configure, and build the code in various environments, including the lab, staging, and production environment.
 
-DataMiner Project Collaboration is tailored specifically towards the deployment of complex ICT media and broadband solutions and fully aligns with agile concepts. It features built-in workflows to ensure that the joint project squad can efficiently collaborate in a distributed constellation and focus their energy on maximizing the value delivery.
+This process includes using **Git for push and pull requests** and utilizing a CI/CD pipeline to **automatically trigger and execute multiple steps**, such as building the solution, executing unit tests, performing code analysis, running the DIS validator, publishing a new version of the protocol, and deploying automation scripts from a GitHub repository.
+
+**Continuous deployment** involves creating a DCP key and adding it as a secret in the repository and adding the Skyline DataMiner Deploy Action to your workflow to streamline the deployment process.
 
 > [!TIP]
-> See also: [Project Collaboration](xref:Collaboration)
+> See also:
+>
+> - [CI/CD Expert Hub](https://community.dataminer.services/expert-hub-ci-cd/)
+> - [CI-CD – Our technical implementation & choice of tools](https://community.dataminer.services/ci-cd-our-technical-implementation-choice-of-tools/)
+> - [CI/CD and the Agile Principles](https://community.dataminer.services/ci-cd-and-the-agile-principles/)
+> - [Empower your DevOps engineers with the DataMiner CI/CD infrastructure](https://community.dataminer.services/empower-your-devops-engineers-with-the-dataminer-ci-cd-infrastructure/)
+> - [CI/CD using DIS, the Cloud Platform and GitHub actions](https://community.dataminer.services/video/ci-cd-using-dis-the-cloud-platform-and-github-actions/)
+> - [CI/CD User Guide](https://docs.dataminer.services/develop/TOOLS/CICD/DevelopmentWithCICD.html)
```

Comment:
I will remove this link for now, as the section you link to is mainly related to internal CI/CD procedures and has little relevance for customers. We're planning to add a new CI/CD section with all the information relevant for them soon, and then a link to that section can be added.